install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/defaults/main.yml

```diff
@@ -14,5 +14,6 @@ securedrop_app_rsync_opts:
   - "--exclude=*.git"
   - "--exclude=*aths"
   - "--exclude=*.deb"
+  - "--exclude=*update_python_requirements"
```

Comment:
This should be omitted entirely based on the reworking of this role to use the `synchronize` module's `.rsync-filter` file support.